draft-ietf-quic-transport.md

```diff
@@ -1483,10 +1483,13 @@ subsequent sections.
 ## STREAM Frame {#frame-stream}
 
 STREAM frames implicitly create a stream and carry stream data. The type byte
-for a STREAM frame contains embedded flags, and is formatted as `11FSSOOD`.
+for a STREAM frame contains embedded flags, and is formatted as `1UFSSOOD`.
 These bits are parsed as follows:
 
-* The first two bits must be set to 11, indicating that this is a STREAM frame.
+* The first bit must be set to 1, indicating that this is a STREAM frame.
+
+* 'U' is the UNI bit, which is used to indicate this is a unidirectional
+  stream, and no stream frames may be sent in response.
```

Comment:
If we went this route, I think the best way to do it would be to think of the stream ID space as being divided into 4 distinct spaces, as opposed to the two we have today.

Today: client | server
Future: client uni | client bidi | server uni | server bidi

In this case, each one of those 4 spaces could have different steam limits (aka, max stream ID).